assets/src/edit-story/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -19,25 +19,47 @@
  */
 import PropTypes from 'prop-types';
 import { __ } from '@web-stories-wp/i18n';
+import { v4 as uuidv4 } from 'uuid';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { Switch } from '../../../form';
+import { Row as DefaultRow } from '../../../form';
 import { SimplePanel } from '../../panel';
 import { getCommonValue } from '../../shared';
+import { Checkbox, Text, THEME_CONSTANTS } from '../../../../../design-system';
+
+const Row = styled(DefaultRow)`
+  justify-content: flex-start;
+  margin-top: 2px;
```

Comment:
Why is the extra margin needed? Shouldn't this be the right structure to ensure proper alignment?

Also, the `justify-content: flex-start` declaration [can be set](https://github.com/google/web-stories-wp/blob/9b09120bdd886b135d465f1ef449fec8f9c8cc79/assets/src/edit-story/components/form/row.js#L25) (or rather, not overwritten) via the `spaceBetween={false}` property

Reply:
In this specific case, it needs more margin since otherwise the animation when clicking on the checkbox still gets hidden partially. Hm, I could also change the margins of the title and the content again -- perhaps that makes more sense.